1-js/02-first-steps/09-comparison/1-comparison-questions/solution.md

```diff
@@ -9,13 +8,12 @@ undefined === null → false
 null == "\n0\n" → false
 null === +"\n0\n" → false
 ```
+कुछ कारण:
 
-Some of the reasons:
-
-1. Obviously, true.
-2. Dictionary comparison, hence false. `"a"` is smaller than `"p"`.
-3. Again, dictionary comparison, first char of `"2"` is greater than the first char of `"1"`.
-4. Values `null` and `undefined` equal each other only.
-5. Strict equality is strict. Different types from both sides lead to false.
-6. Similar to `(4)`, `null` only equals `undefined`.
-7. Strict equality of different types.
+1. जाहिर है, सच है।
+2. शब्दकोश की तुलना, इसलिए झूठी है। `"a"` `"p"` से छोटा है।
+3. फिर से, शब्दकोश तुलना, `"2"` का पहला करैक्टर `"1"` के पहले करैक्टर से अधिक है।
+4. मान `null` और `undefined` एक-दूसरे के बराबर हैं।
+5. सख्त समानता सख्त है। दोनों तरफ से अलग-अलग प्रकार झूठे होते हैं।
```

Comment:
This sentence can be improved. The second sentence in particular needs some changes. A good test is to come back to the article after 30 minutes and see if every line is easy to understand. Translation doesn't need to be exact but it should convey the topic easily.

Reply:
/done